metadata/lang/es.yml

```diff
@@ -24,11 +24,34 @@ supporting-galaxies: "Disponible en estas instancias de Galaxy"
 last-modification: "Última modificación"
 
 
-# other boxes
+# boxes
 tip: "Consejo"
-handson: "Práctica"
+hands-on: "Práctica"
+agenda: "Agenda"
+code-in: 'Entrada'
+code-out: 'Salida'
+comment: 'Comentario'
+details: 'Detalles'
+solution: 'Solución'
+warning: '¡Precaucion!'
+question: 'Preguntas'
+
+# contributions
```

Comment:
@nomadscientist would be great if you could get some spanish translations for these contribution terms.